google-cloud-asset/synth.py

```diff
@@ -133,5 +133,16 @@ def escape_braces(match):
         ])
     )
 
+s.replace(
+    'google-cloud-asset.gemspec',
+    '"README.md"',
```

Comment:
Generally, these synth hacks are to be present until we fix it upstream, so I've generally been writing them so that when it does get fixed upstream, it doesn't break. For example, instead of replacing `"README.md"` with `"README.md", "AUTHENTICATION.md"`, I'd replace `"README.md", "LICENSE"` with `"README.md", "AUTHENTICATION.md", "LICENSE"`

Reply:
Updated.